help/configure-email-visibility.md

```diff
@@ -9,6 +9,14 @@ for new users in the organization.
 
 {start_tabs}
 
+{tab|on-sign-up}
+
+1. After confirming your email, click **Change** below your email address.
+
+1. Configure **Who can access your email address**, and click **Confirm**.
```

Comment:
We usually break them up into two steps. So maybe something like:
```suggestion
1. Configure **Who can access your email address**.
1. Click **Confirm** to apply the email visibility change.
```

Reply:
Sounds good -- I moved confirmation to its own step. Screenshot updated above.